src/main/java/test2/Main.java

```diff
@@ -0,0 +1,71 @@
+package test2;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static java.util.stream.IntStream.rangeClosed;
+
+public class Main {
+
+        public static List<Integer> getNumbersUsingIntStreamRange(int a, int b) {
+            return IntStream.range(a, b+1)
+                    .boxed()
+                    .collect(Collectors.toList());
+        }
+
+        public static List<Integer> rangeNumber (int a, int b) {
```

Comment:
pls rename the method to a more appropriate way
also, what do you think about the argument's name should we rename it as well?